docs/news.md

```diff
@@ -2,11 +2,9 @@
 
 We do not manage release notes.
 
-However you can see a kind of release notes by below `git` command on master branch.
+As a workaround, you can create a "poor man's changelog" via `git` from the master branch:
```

Comment:
What is the expression "poor man's changelog" ? lol Is there the source article or dictionary page explaining this?

Reply:
I want to avoid using any sexual related expression like "man", "woman", "male" and "female" in documents to avoid a risk.

Reply:
I've made the "poor man's changelog" up. A "poor man's $something" is usually referred to something cheaper than the "actual" thing (see for instance: https://www.macmillandictionary.com/dictionary/british/a-poor-man-s-something). And since this is not a proper changelog, but just a simple substitute I thought it might fit. However if you would prefer not to have it here, I can simply drop it.

FYI, I've seen comparable names for instance in the "poor man's bold" LaTeX package, or the [Poor man’s box drawing characters](https://ctan.org/pkg/pmboxdraw) package.

Reply:
I went ahead and changed it to something else.

Reply:
Thanks for the changing. I would prefer the change.

Reply:
Thanks for the source article! I am learning!